GUACAMOLE-966: Bump client version numbers to 1.2.0

necouchman opened the pull request:
Pull Request to bump versions within the client up to 1.2.0.  I did *NOT* bump the API version (`guacamole-common-js/src/main/webapp/modules/Version.js`) as I do not think the API has changed??  Let me know if that one should still be added, though.

mike-jumper commented:
I think the guacamole-common-js API has actually changed for this release. It's not changed in an incompatible way, but I can find at least the `drawStream()` of `Guacamole.Display` as a new addition (added via commit 4e130d2afd as part of [GUACAMOLE-884](https://issues.apache.org/jira/browse/GUACAMOLE-884)).

mike-jumper commented on guacamole-common/pom.xml:
```diff
@@ -26,7 +26,7 @@
     <artifactId>guacamole-common</artifactId>
     <packaging>jar</packaging>
-    <version>1.1.0</version>
+    <version>1.2.0</version>
```
Has guacamole-common changed? I'm not seeing any changes from 1.1.0 according to git.

necouchman replied:
Good point - reverted that.

mike-jumper commented on extensions/guacamole-auth-cas/pom.xml:
```diff
@@ -210,15 +210,15 @@
             <groupId>org.apache.guacamole</groupId>
             <artifactId>guacamole-common</artifactId>
-            <version>1.1.0</version>
+            <version>1.2.0</version>
```
It might be easier to just remove this dependency, relying on inheriting it from guacamole-ext. Unlike projects like guacamole-example, use of guacamole-common within an extension will be to service guacamole-ext.

necouchman replied:
Removed.

mike-jumper commented on extensions/guacamole-auth-radius/pom.xml:
```diff
@@ -178,15 +178,15 @@
             <groupId>org.apache.guacamole</groupId>
             <artifactId>guacamole-common</artifactId>
-            <version>1.1.0</version>
+            <version>1.2.0</version>
```
Same here:

It might be easier to just remove this dependency, relying on inheriting it from guacamole-ext. Unlike projects like guacamole-example, use of guacamole-common within an extension will be to service guacamole-ext.

necouchman replied:
Removed.

necouchman commented:
Okay - addressed the issues:
* Bumped the JS API version up to 1.2.0.
* Reverted the common version to 1.1.0
* Removed the explicit dependency on `guacamole-common` from `pom.xml` files where `guacamole-ext` is already being pulled in.

mike-jumper commented on doc/guacamole-example/pom.xml:
```diff
@@ -106,15 +106,15 @@
             <groupId>org.apache.guacamole</groupId>
             <artifactId>guacamole-common</artifactId>
-            <version>1.1.0</version>
+            <version>1.2.0</version>
```
1.1.0*

necouchman replied:
Oops, fixed.
